lib/improver/tests/generate_ancillaries/test_generate_ancillary.py

```diff
@@ -46,6 +46,7 @@
 
 
 def _make_test_cube(long_name, stash=None):
```

Comment:
As mentioned above, I think a unit test of the behaviour where you pass in a 1 or 3 values for the topographic bands would be good e.g. `topographic_bounds=[10]` and `topographic_bounds=[10, 20, 30]`.